wandb/sdk/launch/_project_spec.py

```diff
@@ -142,7 +138,7 @@ def image_name(self) -> str:
         """Returns {PROJECT}_launch the ultimate version will
         be tagged with a sha of the git repo"""
         # TODO: this should likely be source_project when we have it...
-        return f"{self.target_project}_launch"
+        return f"{self.target_project.lower()}_launch"
```

Comment:
We may want to call .lower() on this when we set the `target_project` attribute.  This is screaming for weird bugs where we forget to call `.lower()`

Reply:
Good point, I'll do that